### DISCUSSION TITLE
Welcome to SurgSim_Lite Discussions!

### DISCUSSION THREAD

**Discussion by razterizer** [razterizer/SurgSim_Lite#4] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by indrawibawa85** [comment]
You should look at processing.org --> Its an IDE to generate graph. Its the IDE used and modified for Adruino IDE studio. But the language is java, have a try! It helped me graduate my master degree ;)

**Reply by razterizer** [reply]
Aha. This reminds me of godbolt.org which I use now and then to do quick C++ experiments. Nice that you can draw geometry and stuff online. Alas a bit too high-tech for the system I've developed here on github though. 😅
